### PR TITLE
Prepare for 0.19 release

### PR DESCRIPTION
**Pull request by andreamari**
### License

- [ x] I license this contribution under the terms of the GNU GPL, version 3 and grant Unitary Fund the right to provide additional permissions as described in section 7 of the GNU GPL, version 3.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,53 @@
...
+- remove `print` calls (#1519) [@natestemen]
+- BQSKit tutorial (#1489) [@natestemen]
+- Update pyquil requirement from ~=3.3.0 to ~=3.3.1 (#1516) [@dependabot[bot]]
+- Update scipy requirement from ~=1.7.3 to ~=1.9.1 (#1467) [@dependabot[bot]]
+- Update mypy requirement from ~=0.971 to ~=0.981 (#1511) [@dependabot[bot]]
+- (partially) upgrade numpy (#1501) [@natestemen]
+- Update amazon-braket-sdk requirement from ~=1.30.2 to ~=1.31.0 (#1512) [@dependabot[bot]]
+- Add whitepaper publication info/links (#1456) [@natestemen]
+- Support python 3.9 (#1505) [@natestemen]
+- Learning function for biased noise (#1358) [@misty-w]
```
Comment (Misty-W) [inline CHANGELOG.md:36]:
```suggestion
- Learning function for biased noise (#1358) [@Misty-W]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,53 @@
...
+## Version 0.19.0
+
+### Summary
+
+With this release Mitiq supports most recent versions of Python: `3.8`, `3.9` and `3.10`!
+We drop support for Python `3.7`.
+
+Mitiq is now compatible with Numpy `1.21.6`. Different versions of NumPy may not work properly.
+
+Another important update is the addition of new [tools](https://mitiq.readthedocs.io/en/latest/apidoc.html#module-mitiq.pec.representations.biased_noise) for applying learning-based PEC!
```
Comment (Misty-W) [inline CHANGELOG.md:12]:
```suggestion
Another important update is the addition of new tools for applying learning-based PEC!
This release introduces a function for learning depolarizing noise representations from Clifford circuit data. 
Read more in our updated [API-doc](https://mitiq.readthedocs.io/en/latest/apidoc.html#module-mitiq.pec.representations.biased_noise).
```

**Review by Misty-W — APPROVED** [review]
Slightly expanded explanation of learning-based PEC and fixed the case of my username. Everything else LGTM!

**Review by natestemen — APPROVED** [review]
LGTM! :shipit:

File: CHANGELOG.md (hunk 1 above)
Comment (andreamari) [reply CHANGELOG.md]: Oh, sorry for this. I copied it from an old changelog, where I probably did the same mistake :disappointed:

**Pull request merged**